### PR TITLE
Basic Search Feature - Google News RSS Feed

### PR DESCRIPTION
**Pull request by avinashraja98**
The bot will take in anything sent to it and query it at google news which will respond with a rss feed. This is parsed and sent as a response.

### CODE REVIEW

**Comment by SteveParson** [comment]
Thanks for your contribution! While I am not familiar enough with node.js to review your code, I would suggest attributing https://github.com/apollonian11/reactriot2017-reaction/blob/master/serverApi.js directly in the file that uses it. I am sure that when @moustafaelsisy returns he will check the code out.

Thanks again!

**Comment by nint8835** [comment]
I agree with what Steve suggested. In addition:
* Most of the stuff added to the gitignore is not required. You could probably chop out pretty much everything besides the solution files.
* We already have a setup guide in the readme, and as far as I can tell, your usage.md is the same thing, so it isn't required
* I would suggest adding the line `This bot returns the top hit from google news related to the topic sent.` into the readme instead of having it in it's own separate file

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,267 @@
...
+.idea/
+*.sln.iml
+
+# CodeRush
+.cr/
+
+# Python Tools for Visual Studio (PTVS)
+__pycache__/
+*.pyc
+
```
Comment (nint8835) [inline .gitignore]: Most of the above changes to the gitignore are not required. Really all you need are the solution files.

File: notes.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+This bot returns the top hit from google news related to the topic sent.
```
Comment (nint8835) [inline notes.txt]: I would suggest moving this line into the readme, instead of having it's own file

File: notes.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+searchArticle.js is from the repo
```
Comment (nint8835) [inline notes.txt]: searchArticle.js is licensed under MIT, which requires you to include it's license and copyright information. I would suggest creating a file along the lines of LICENSE_searchArticle.js and copying the contents of [this file](https://github.com/apollonian11/reactriot2017-reaction/blob/master/LICENSE) into it

File: usage.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  mv configBlueprint.js config.js
+ 
+5. Add token to config.js using preferred text editor
+
+  nano config.js
+  
+6. run bot.js
+
+  node bot.js
+
```
Comment (nint8835) [inline usage.md]: The readme file already has installation instructions, so this file is redundant. If you have any changes that need to be made, I would suggest you move them there.

**Review by nint8835 — CHANGES_REQUESTED** [review]
Just a few changes need to be made and then it will be ready to be merged

**Comment by avinashraja98** [comment]
Changes made as requested. Thanks for the input. Please comment if anything else is required.

**Review by nint8835 — APPROVED** [review]

**Pull request merged**